lib/index.js

```diff
@@ -0,0 +1,108 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const assert = require('assert-plus');
+const bedrock = require('bedrock');
+const {util: {clone, BedrockError}} = bedrock;
+
+const packages = [];
+
+/**
+ * @module bedrock-package-manager
+ */
+
+/**
+ * Get package information.
+ *
+ * @param {Object} options - The options to use.
+ * @param {string} [options.alias] - The package alias.
```

Comment:
Why would we need an alias? Should using only the package name suffice?

Reply:
@gannan08 I answered that question here: https://github.com/digitalbazaar/bedrock-ssm-mongodb/pull/1#discussion_r267471192

Is that adequate.  And `alias` is optional, so there is no requirement that it be used.